### ISSUE TITLE
Some API calls fail with ECONNRESET on some machines

### ISSUE THREAD

**Issue by avahowell** [NebulousLabs/Sia-UI#392]
Users have occasionally been reporting connection errors such as `ECONNRESET` in their UI, indicating that some Siad API calls fail.  Not sure exactly what's causing this, but it seems to crop up on Windows more than other platforms.

**Comment by DavidVorick** [comment]
Is there some way that we can gracefully detect these errors in the UI and the re-establish the connection without reporting anything to the user (assuming that the only UX penalty is an occasional few ms added to the user request time)

**Comment by avahowell** [comment]
yes, we could use an exponential backoff like we did in the upnp package.

**Comment by lukechampine** [comment]
has this been resolved by longer request timeouts?
